lib/styles/components/_buttons.scss

```diff
@@ -3,48 +3,177 @@
 @import '../settings/spacings';
 @import '../settings/radiuses';
 @import '../settings/typography';
+@import '../settings/buttons';
 
-$default-lighter-shadow: 0 4px 12px rgba($color-firefly-black, 0.35);
```

Comment:
do we need to import old `colors` in this file?

Reply:
Fixed.